nemo/collections/asr/models/ctc_models.py

```diff
@@ -173,8 +175,8 @@ def training_step(self, batch, batch_nb):
         loss_value = self.loss(
             log_probs=log_probs, targets=transcript, input_lengths=encoded_len, target_lengths=transcript_len
         )
-        wer = self.__wer(predictions, transcript, transcript_len)
-        tensorboard_logs = {'train_loss': loss_value, 'training_wer': wer}
+        wer_num, wer_denum = self.__wer(predictions, transcript, transcript_len)
```

Comment:
Same nitpick as before (denum -> denom), here and in the following changed lines